mainapp/templates/account.html

```diff
@@ -60,6 +78,97 @@ <h4 class=" mb-0">
         </div>
     </div>
 
+    <!-- Landlord Apartments -->
+    {% if user.landlord_apartment_set %}
```

Comment:
I think this is still displaying, even if the set is empty. Could we check if there are no elements in the list and not display if so?  

![image](https://user-images.githubusercontent.com/32490273/69197230-dd5c3500-0afe-11ea-9c2e-d752a7dc4aa8.png)

Reply:
Good catch! I guess `user.landlord_apartment_set` is always there so we need to explicitly check for `user.landlord_apartment_set.count>0`.